src/darwin/Framework/CHIP/CHIPOperationalCredentialsDelegate.mm

```diff
@@ -43,19 +43,25 @@ static BOOL isRunningTests(void)
     return (environment[@"XCTestConfigurationFilePath"] != nil);
 }
 
-CHIP_ERROR CHIPOperationalCredentialsDelegate::init(CHIPPersistentStorageDelegateBridge * storage)
+CHIP_ERROR CHIPOperationalCredentialsDelegate::init(CHIPPersistentStorageDelegateBridge * storage, ChipP256KeypairPtr nocSigner)
 {
     if (storage == nil) {
         return CHIP_ERROR_INVALID_ARGUMENT;
     }
 
+    CHIP_ERROR err = CHIP_NO_ERROR;
     mStorage = storage;
 
-    CHIP_ERROR err = LoadKeysFromKeyChain();
+    if (!nocSigner) {
```

Comment:
It does not seem immediately obvious here that both sides initialize mIssuerKey.
How about extracting the `mIssuerKey.reset(new chip::Crypto::P256Keypair());` here and let LoadKeysFromkeyChain/GenerateKeys use mIssuerKey assuming it exists?(

Why do both LoadKeysFromKeyChain and GenerateKeys reset mIssuerKey to new values? old code used to just reuse the same object.

Reply:
Yeah that is cleaner for sure, thank you!